webfonts/README.md

```diff
@@ -2,17 +2,69 @@
 
 ## Font file CSS
 
-All static fonts are available via `./red-hat-font.css`
+Original fonts are available via `./font.css`
```

Comment:
I feel like this is a dumb question.. but I'm not sure where to find this file.

Reply:
within the webfonts folder, but that's not super obvious. Let me update it